5.6/config/.drush/drush.yml

```diff
@@ -0,0 +1,5 @@
+options:
+  uri: 'http://${env.VIRTUAL_HOST}'
+  root: '/var/www/${env.DOCROOT}'
+ssh:
+  options: '-o LogLevel=QUIET -t '
```

Comment:
Why is `-t` necessary?

```
-t
Forces a tty allocation even if a command is specified. You can also configure this in the configuration file using the ForcePTTYAllocation keyword.
```

Reply:
I've run into issues where if `-t` isn't included then the remote in the alias won't read input properly or at all sometimes.